toolbox/util/githubClient.go

```diff
@@ -37,7 +37,7 @@ const (
 	releaseBodyTemplate = `
 [ARTIFACTS](http://gcsweb.istio.io/gcs/istio-release/releases/{{.ReleaseTag}}/)
 
-[RELEASE NOTES](https://github.com/istio/istio/wiki/({{.ReleaseTag}})`
+[RELEASE NOTES](http://github.com/istio/istio/wiki/v{{.ReleaseTag}})`
```

Comment:
why using http instead of https?

Reply:
https does work well currently with gcsweb.